pom.xml

```diff
@@ -1482,6 +1482,17 @@
             <name>Maven Repository</name>
             <url>https://repo.maven.apache.org/maven2</url>
         </repository>
+
+        <repository>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>kyuubi-shaded-staging-0.4.0-rc0</id>
+            <url>https://repository.apache.org/content/repositories/orgapachekyuubi-1045</url>
```

Comment:
I just finished the release of Kyuubi Shaded 0.4.0, Jar will be available on Maven Central soon, the staging repo could be removed now.